packages/framework/tests/Unit/Model/Order/Item/OrderItemTest.php

```diff
@@ -110,8 +110,7 @@ public function testProductCannotHaveProduct(): void
     {
         $orderItem = $this->createOrderTransport();
 
-        $this->expectException(WrongItemTypeException::class);
-        $orderItem->hasProduct();
+        $this->assertFalse($orderItem->isTypeProductAndHasProduct());
```

Comment:
It seems that this test method had a weird name. Could you please rename it to something like `testTransportCannotHaveProduct()`? :pray: